templates/diff/templates/sections/added_instance_concepts.html

```diff
@@ -0,0 +1,88 @@
+{% import "macros.html" as mc %}
+
+<h1 class="ui header">Added concepts</h1>
+<section class="ui basic segment">
+    <p>The table below lists the <strong>added concepts</strong>
+    </p>
+    {% set select_query =
+    "prefix owl: <http://www.w3.org/2002/07/owl#>
+prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+prefix xsd: <http://www.w3.org/2001/XMLSchema#>
```

Comment:
are the prefixes used here "enough" for all possible queries?

Reply:
This is a valid comment. Mostly yes, for the context of EuVocabualries team. However, if we decide to move towards a more automated manner of generating the SPARQL queries and the diff report templates, then, prefix management falls quickly within the scope of that generator.